src/testUtils/mocks/dataSources.ts

```diff
@@ -317,8 +317,8 @@ interface ContextOptions {
   getIdentity?: Identity;
   getIdentityClaimsFromChain?: ClaimData[];
   getIdentityClaimsFromMiddleware?: ResultSet<ClaimData>;
-  primaryAccount?: string;
-  secondaryAccounts?: SecondaryAccount[];
+  primaryAccount?: PermissionedAccount;
```

Comment:
Considering this will always have the same permissions, I'd change it to be `string` again and just wrap it in an object with permissions inside the configure method to avoid having to write the same permissions over and over in tests